zagar_server/src/main/java/accountserver/AccountServer.java

```diff
@@ -58,7 +57,12 @@ public static void main(@NotNull String[] args) throws Exception {
   public void run() {
     startApi();
     while (true) {
-      ApplicationContext.instance().get(MessageSystem.class).execForService(this);
+      try {
+        ApplicationContext.instance().get(MessageSystem.class).execOneForService(this, 100);
```

Comment:
`ClientConnectionService` accepts messages from `Mechanics` and then does replication. On each iteration it tries to poll message from queue. If there are some messages in queue it executes them consequently _without any timeout_. In case of empty queue it waits 100 ms for message and then goes further (to the next iteration of while block).

One can set timeout to 100 sec (or even call `execOneForService()`) and log each `ReplicateMsg` executing. The result doesn't depend on what timeout was set.

>2016-11-27 19:22:03,507 DEBUG [client_connection_service] messages.ReplicateMsg (ReplicateMsg.java:27) - **ReplicationMsg exec() call**
2016-11-27 19:22:04,507 INFO  [mechanics] ticker.Ticker (Ticker.java:42) - [mechanics] <tick> 2
2016-11-27 19:22:04,508 INFO  [mechanics] mechanics.Mechanics (Mechanics.java:38) - Start replication
2016-11-27 19:22:04,508 INFO  [mechanics] ticker.Ticker (Ticker.java:37) - All tickers finish at 0 ms
2016-11-27 19:22:04,508 DEBUG [client_connection_service] messages.ReplicateMsg (ReplicateMsg.java:27) - **ReplicationMsg exec() call**
2016-11-27 19:22:05,508 INFO  [mechanics] ticker.Ticker (Ticker.java:42) - [mechanics] <tick> 3
2016-11-27 19:22:05,508 INFO  [mechanics] mechanics.Mechanics (Mechanics.java:38) - Start replication
2016-11-27 19:22:05,508 INFO  [mechanics] ticker.Ticker (Ticker.java:37) - All tickers finish at 0 ms
2016-11-27 19:22:05,509 DEBUG [client_connection_service] messages.ReplicateMsg (ReplicateMsg.java:27) - **ReplicationMsg exec() call**
2016-11-27 19:22:06,509 INFO  [mechanics] ticker.Ticker (Ticker.java:42) - [mechanics] <tick> 4
2016-11-27 19:22:06,509 INFO  [mechanics] mechanics.Mechanics (Mechanics.java:38) - Start replication
2016-11-27 19:22:06,509 INFO  [mechanics] ticker.Ticker (Ticker.java:37) - All tickers finish at 0 ms
2016-11-27 19:22:06,509 DEBUG [client_connection_service] messages.ReplicateMsg (ReplicateMsg.java:27) - **ReplicationMsg exec() call**
2016-11-27 19:22:07,509 INFO  [mechanics] ticker.Ticker (Ticker.java:42) - [mechanics] <tick> 5
2016-11-27 19:22:07,509 INFO  [mechanics] mechanics.Mechanics (Mechanics.java:38) - Start replication
2016-11-27 19:22:07,510 INFO  [mechanics] ticker.Ticker (Ticker.java:37) - All tickers finish at 0 ms
2016-11-27 19:22:07,510 DEBUG [client_connection_service] messages.ReplicateMsg (ReplicateMsg.java:27) - **ReplicationMsg exec() call**